terra/models.py

```diff
@@ -247,6 +247,11 @@ def total_days_out(self):
     def approved_total(self):
         return utils.format_currency(self.total_funding())
 
+    def total_per_fund(self):
```

Comment:
@aprigge This new method `total_per_fund` doesn't seem to be used anywhere.  It also looks like it returns a list of funds, not totals.  I recommend removing it, or renaming to make its purpose clearer if we know it will be needed in the near future.